Description: Set statsdir by default
 This avoids the need to explicitly set "statsdir" in /etc/ntpsec/ntp.conf.
 .
 This is a change from the ntp ("NTP Classic") package, which requires the
 statsdir to be explicitly set for logging to happen.  However, that is only
 happening because /var/NTP does not normally exist, so it is a bit of an
 accidental feature.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@debian.org>
Last-Update: 2021-09-25
--- a/docs/includes/ntpd-body.adoc
+++ b/docs/includes/ntpd-body.adoc
@@ -505,7 +505,7 @@
 |process ID file         |none              |+-p+        |+pidfile+
 |log file                |system log        |+-l+        |+logfile+
 |include file            |none              |none        |+includefile+
-|statistics path         |+/var/NTP+        |+-s+        |+statsdir+
+|statistics path         |+/var/log/ntpsec+ |+-s+        |+statsdir+
 |keys file               |none              |+-k+        |+keys+
 |===================================================================
 
--- a/ntpd/ntp_util.c
+++ b/ntpd/ntp_util.c
@@ -50,7 +50,7 @@
  * Statistics file stuff
  */
 #ifndef NTP_VAR
-# define NTP_VAR "/var/NTP/"		/* NOTE the trailing '/' */
+# define NTP_VAR "/var/log/ntpsec/"		/* NOTE the trailing '/' */
 #endif
 
 
